src/protect-api/Program.cs

```diff
@@ -1,26 +1,37 @@
+// <ms_docref_import_types>
 using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.Identity.Web;
+// </ms_docref_import_types>
 
 var builder = WebApplication.CreateBuilder(args);

```

Comment:
I'll be creating a style guide and these will go in it - this is good stuff to work through!

- ✔️ **DO include object initialization for critical types** - A common complaint we receive is that we don't include initialization of critical types in our snippets. As such, devs don't know how to get the object the sample is calling a method on. Comments like:

    > `app.DoSomething()`
    > `app.DoSomethingElse()`
    > 
    > WTF is `app`, and how do I get it?!

- ❌ **DON'T use type inference on object initialization** - Avoid `var` on instance initializations so its type is super clear, especially on things like builder classes where the type name of the object you get back differs from the builder. Though compiler can infer the type, the reader can't when looking at the code on a page. 😉

Reply:
thanks for the feedback @mmacy,

done | addressed from [d0138](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/3/commits/d01386020c4844e5ab61211cfe6f43d317177f8a) and [3f035](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/3/commits/3f0350ddc0a2d74daa8ad90e66c56c64502d7b75)